bundles/io.github.linkedfactory.service/src/main/java/io/github/linkedfactory/service/config/KvinPartitionedFactory.java

```diff
@@ -0,0 +1,48 @@
+package io.github.linkedfactory.service.config;
+
+import io.github.linkedfactory.core.kvin.Kvin;
+import io.github.linkedfactory.core.kvin.partitioned.KvinPartitioned;
+import net.enilink.composition.annotations.Iri;
+import org.eclipse.core.runtime.Platform;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+@Iri("plugin://io.github.linkedfactory.service/data/KvinPartitioned")
+public abstract class KvinPartitionedFactory implements IKvinFactory{
+    private static final Logger log = LoggerFactory.getLogger(KvinPartitionedFactory.class);
+
+    @Override
+    public Kvin create() {
+        try {
+            String path = getArchivePath();
+            Integer cycle = getCycle();
+            if (path == null) {
+                path = "partition";
+            }
+            if(cycle == null) {
```

Comment:
Please fix the formatting by using IDEA autoformat.
Maybe we should provide a formatter config file for IDEA and Eclipse or something like the Maven-based formatting in RDF4J:
https://github.com/eclipse-rdf4j/rdf4j/blob/70344922d0825fe8224e515c70d7367c6f90ffde/pom.xml#L268